django/chat/templates/chat/components/chat_list_item_menu.html

```diff
@@ -18,7 +18,7 @@
     <h6 class="dropdown-header" role="presentation">{% trans 'Options' %}</h6>
     <div class="list-group list-group-flush options-menu">
       <button type="button"
-              hx-get="{% url 'chat:rename_chat' chat_id=chat.id current_chat=chat.current_chat %}"
+              hx-get="{% url 'chat:rename_chat' chat_id=chat.id section=chat.section current_chat=chat.current_chat %}"
```

Comment:
I don't really understand why you need to keep track of the chat section numbers?